### DISCUSSION TITLE
Trojans detected in latest install

### DISCUSSION THREAD

**Discussion by cmull123** [dbeaver/dbeaver#35060] [General]
I had a user attempting to install dbeaver, was flagged by ThreatLocker as a trojan.  Is this a known issue?  I can't find any reference other than a known false positive back in 2018 with different signatures.  This was downloaded from dbeaver.io

c:\users\threatlocker\appdata\local\dbeaver\plugins\org.eclipse.equinox.p2.repository.tools_2.4.400.v20240416-0654.jar VirIT
c:\users\threatlocker\appdata\local\dbeaver\swt-win32-4965r11.dll
c:\users\threatlocker\appdata\local\dbeaver\plugins\org.eclipse.ui.intro.nls_3.7.400.v20240414-0828.jar
c:\users\threatlocker\appdata\local\dbeaver\plugins\org.eclipse.ui.navigator.resources.nls_3.9.300.v20240506-0929.jar
c:\users\threatlocker\appdata\local\dbeaver\plugins\org.eclipse.ui.console.nls_3.14.100.v20240429-1358.jar

**Comment by E1izabeth** [comment]
@cmull123 have you downloaded portable version?
We have similar issue with another anti-virus app https://github.com/dbeaver/dbeaver/issues/34445

**Comment by ShadelessFox** [comment]
I see why it might complain about `swt-win32-4965r11.dll`, a dynamic library with executable code. But marking `nls` JAR files seems quite weird to me - these contain nothing but localization resources stored in plaintext format.

**Comment by serge-rider** [comment]
It is clearly a false positive. 
But I couldn't find where we can report about it to ThreatLocker
